test/bondingEvent.js

```diff
@@ -26,11 +26,15 @@ describe('BondingEvent', async () => {
     RatioCalculator = await RatioCalculatorContract.deploy();
   });
 
+  const isSEuroToken0 = () => {
+    return SEuro.address.toLowerCase() < USDT.address.toLowerCase();
```

Comment:
you would not believe how long i was trying to figure out what was going on with this bug 😭 
and it only failed when running the whole test suites (because that affects which contracts receive which address strings) ⏳ ⏲️